classes/ActionScheduler_wpPostStore.php

```diff
@@ -627,14 +627,7 @@ protected function get_actions_by_group( $group, $limit, DateTime $date ) {
 			),
 			'date_query'       => array(
 				'column' => 'post_date',
-				array(
-					'compare' => '<=',
-					'year'    => $date->format( 'Y' ),
-					'month'   => $date->format( 'n' ),
-					'day'     => $date->format( 'j' ),
-					'hour'    => $date->format( 'G' ),
-					'minute'  => $date->format( 'i' ),
-					'second'  => $date->format( 's' ),
+				'before' => $date->format( 'Y-m-d H:i:s' ),
 				),
```

Comment:
This `),` needs to go too now the array has been deleted. This is leading to Travis failures.